### ISSUE TITLE
Final Training Loss is still around 2.xxx

### ISSUE THREAD

**Issue by fansia** [2017-fall-DL-training-program/ImageCaption#20]
Hi TAs,

I found the final training losses of show_attend_tell and topdown models are still around 2.1xxx.
That is much higher than previous lab(ResNet. final traning loss could be under 0.1 )

total num of epoch = 5

1) Is that right? 
2) If it is right, why the losses are still high?

Thank you.
Eric

**Comment by connie980149** [comment]
1. That is right.
2. Your loss is the distance between the target sentence and the generated sentence. Because this is not classification problem, the answer is not determinate (like the dates: 1 image has 5 captions, it has many answers). If your loss is under 0.1 that means the model is overfitting the data.
